src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -397,26 +397,9 @@ This command prepares the underlying cloud infrastructure for Submariner install
 <!-- markdownlint-disable line-length -->
 | Flag                             | Description
 |:---------------------------------|:--------------------------------------------------------------------------------------------------------------------------|
-| `--as` `<string>`                       | Username to impersonate for the operation
-| `--as-group` `<stringArray>`            | Group to impersonate for the operation, this flag can be repeated to specify multiple groups
-| `--as-uid` `<string>`                   | UID to impersonate for the operation
-| `--certificate-authority` `<string>`    | Path to a cert file for the certificate authority
-| `--client-certificate` `<string>`       | Path to a client certificate file for TLS
-| `--client-key` `<string>`               | Path to a client key file for TLS
-| `--cluster` `<string>`                  | The name of the kubeconfig cluster to use
-| `--context` `<string>`                  | The name of the kubeconfig context to use
 | `--insecure-skip-tls-verify` `<string>` | If true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure
```

Comment:
This is a connection flag too.

Reply:
@sridhargaddam's original issue (https://github.com/submariner-io/submariner-website/issues/848) requested in particular `--insecure-skip-tls-verify` as a flag that should be added, so CCing him.

Reply:
Right, the same flag is supported even for `subctl join ...` command as well. I'm fine if you want to include it in this PR or as a separate commit/PR. Thanks for looking into it @dfarrell07